src/index.js

```diff
@@ -42,6 +44,7 @@ const plugin = {
 				"json/no-empty-keys": "error",
 				"json/no-unsafe-values": "error",
 				"json/no-unnormalized-keys": "error",
+				"json/top-level-interop": "off",
```

Comment:
```suggestion
```

I think we should just omit it from the recommended config, as disabling it sounds like "it's recommended not to use this rule". @nzakas what do you think?